08.decoding.process.md

```diff
@@ -6664,6 +6664,9 @@ For example, a real implementation might use these arrays to display the frame t
 
 ##### Intermediate output preparation process
 
+The output of this process are the variables w, h, subX, and subY describing the format of the data in arrays OutY, OutU, 
+and OutV.
```

Comment:
The output values are normally specified as the final sentence in the process.

Reply:
The line where this process is invoked:

"The intermediate output preparation process specified in [section 7.18.2][] is invoked to prepare arrays OutY, OutU, and OutV."

should say what happens to the output variables, e.g.

", and the outputs are assigned to w, h, subX, and subY."

Reply:
I added the text to where the function is invoked. 

I have seen a lot of places in the spec, where the output variables were both in the beginning and in the end of the function. I added the sentence in the end as well. 

Since these are purely editorial changes, feel free to modify my text directly if needed.